### PR TITLE
Added Build-BicepParameterFile and New-BicepParameterFile

### PR DESCRIPTION
**Pull request by PalmEmanuel**
As discussed in #28 I've added two new commands.

There is still missing help documentation as well as some optimization which should probably be done in connection with #25 to make sure they are all generated in the same way.

@StefanIvemo check it out and see if you feel like it's working as expected for now!

### CODE REVIEW

**Comment by PalmEmanuel** [comment]
I also took the opportunity to change some naming and removed whitespace here and there for consistency.

**Comment by StefanIvemo** [comment]
Cool! Will test it first thing tomorrow. We should for sure align this with #25. Maybe could add something like `Build-Bicep -Path nsg.bicep -GenerateParameterFile BicepParams` which will generate both ARM Template parameter file and our bicep style workaround.

File: Source/Public/Build-BicepParameterFile.ps1 [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+            }
+            else {
+                $OutputFilePath = $File.FullName -replace '\.bicep','.parameters.json'
+            }
+
+            if (-not [string]::IsNullOrWhiteSpace($ARMTemplate)) {
+                $ARMTemplateObject = $ARMTemplate | ConvertFrom-Json | Select-Object -Property '$schema',contentVersion,@{
+                    Name = 'parameters'
+                    Expression = { $_.Variables }
+                }
```
Comment (StefanIvemo) [inline Source/Public/Build-BicepParameterFile.ps1:50]: Parameter files have a different schema than ARM templates. It has to be updated here.

`$ARMTemplateObject.'$schema'="https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#"`

**Comment by StefanIvemo** [comment]
Did some testing, appears to be working if altering the schema according to the comment above. Need to do some more testing when my head is in a normal state though. 🤯

**Comment by StefanIvemo** [comment]
@PalmEmanuel I don't like my idea any longer, I've been testing this but it will be to confusing for the users. I'm closing this one ☹

**Pull request closed without merging**